docs/root/configuration/listeners/listener_filters/original_dst_filter.rst

```diff
@@ -3,13 +3,41 @@
 Original Destination
 ====================
 
+Linux
+===============
+
 Original destination listener filter reads the SO_ORIGINAL_DST socket option set when a connection
 has been redirected by an iptables REDIRECT target, or by an iptables TPROXY target in combination
 with setting the listener's :ref:`transparent <envoy_v3_api_field_config.listener.v3.Listener.transparent>` option.
+
+Windows
+===============
+
+Original destination listener filter reads the SO_ORIGINAL_DST socket option set when a connection
+has been redirected by an HNS policy applied to a container endpoint. For this fiter to work the
```

Comment:
Will the reader understand what `HNS` means from context?

Reply:
Spelling: fiter

Reply:
Doesn't hurt to add a link to make sure that it makes sense for everyone, added in 3437dc6078878c8282a466a53d09441d3b8437da